### PR TITLE
Remove command test equals

### LINKED ISSUE #99: List implementation for delivery

### PR DESCRIPTION
**Pull request by Wincenttjoi**
- [x] Test cases for removecommandtest
- [x] Added typical delivery items for tests
- [x] Remove codecov report/ patch for easier workflow
- [x] Integration tests for list-d
- [x] Update UG: Multiple tables for command summary

Changes:
- Added equals test to increase coverage in removeCommandTest
- Changed @halcon-blanco 's RED and GREEN colouring, RED when stock is low (<20%) green between 20-100% and light blue above 100%.
- Refactor inventorybookparser into oneshelfbookparser: its misleading since inventorybookparser contains delivery parsers.

Fixes #99

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/RemoveCommand.java [hunk 1]
```diff
@@ -102,11 +102,6 @@ public boolean equals(Object other) {
             return true;
         }
 
-        // instanceof handles nulls
-        if (!(other instanceof EditCommand)) {
```
Comment (saad-haider-pp) [inline src/main/java/seedu/address/logic/commands/RemoveCommand.java]: shouldn't this be replaced by `if (!(other instanceof RemoveCommand))` instead

File: src/main/java/seedu/address/ui/ItemCard.java [hunk 2]
```diff
@@ -74,10 +74,10 @@ public ItemCard(Item item, int displayedIndex) {
                 quantityStats.setText(stats);
             }
 
-            if (percentage > 100) {
-                quantityStats.setFill(Color.RED);
-            } else if (percentage <= 20) {
+            if (percentage > 20) {
```
Comment (saad-haider-pp) [inline src/main/java/seedu/address/ui/ItemCard.java]: Sorry I think I misunderstood what you asked last time haha. But do you think there should be some indication if stock goes above 100% as well

File: src/test/java/seedu/address/logic/commands/RemoveCommandTest.java [hunk 3]
```diff
@@ -44,4 +47,17 @@ public void execute_itemAcceptedByModel_removeSuccessful() throws CommandExcepti
                 commandResult.getFeedbackToUser());
     }
 
+    @Test
+    public void equals() {
```
Comment (saad-haider-pp) [inline src/test/java/seedu/address/logic/commands/RemoveCommandTest.java]: I believe you should also try a test case with a non RemoveCommand object. I believe the current equals method doesn't handle this and will throw a ClassCastException

File: src/main/java/seedu/address/ui/ItemCard.java (hunk 2 above)
Comment (Wincenttjoi) [reply src/main/java/seedu/address/ui/ItemCard.java]: Sure, I will include a different colour for that as well

File: src/main/java/seedu/address/logic/commands/RemoveCommand.java (hunk 1 above)
Comment (Wincenttjoi) [reply src/main/java/seedu/address/logic/commands/RemoveCommand.java]: Made the necessary changes including the test case

File: src/test/java/seedu/address/testutil/DeliveryBuilder.java [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+
+/**
+ * A utility class to help with building Delivery objects.
+ */
+public class DeliveryBuilder {
+
+    public static final String DEFAULT_DELIVERY_NAME = "Damith";
+    public static final String DEFAULT_PHONE = "91231231";
+    public static final String DEFAULT_ADDRESS = "Jln Parang Tritis No. 92 Orchard Road";
+    public static final String DEFAULT_ORDER = "Fried Rice 1x, Iced Kopi less sugar x1";
```
Comment (Choo-Xing-Yu) [inline src/test/java/seedu/address/testutil/DeliveryBuilder.java:17]: should quantity come before item name?

Comment (saad-haider-pp) [reply src/test/java/seedu/address/testutil/DeliveryBuilder.java:17]: apart from this small issue, it LGTM for me!

**Pull request merged**